test/videojs-http-streaming.test.js

```diff
@@ -3720,7 +3761,7 @@ QUnit.module('HLS Integration', {
     this.env = useFakeEnvironment(assert);
     this.requests = this.env.requests;
     this.mse = useFakeMediaSource();
-    this.tech = new (videojs.getTech('Html5'))({});
+    this.player = createPlayer();
```

Comment:
if you create the player and then set `this.tech` afterwards, you won't need to change all the tests below.

Reply:
That is true, but I decided to follow the convention elsewhere in the tests of referencing the tech as a property of the player when a player exists. Happy to change it if you feel strongly about it.

Reply:
this is fine, carry on 🙂